nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkService.java

```diff
@@ -38,14 +38,14 @@
     /**
      * @return the command status corresponding to the command identifier.
      */
-    BulkStatus getStatus(String bulkId);
+    BulkStatus getStatus(String commandId);
 
     /**
-     * Waits for completion of given bulk.
+     * Waits for completion of given bulk command.
      *
-     * @param timeout the time to wait
-     * @param unit the timeout unit
-     * @return {@code true} if bulk completed, or {@code false} if computation has not finished after the timeout
+     * @param commandId the command to wait
+     * @param duration the duration to wait
```

Comment:
this is a timeout duration, i.e. the maximum time we want to wait for the command completion